### PR TITLE
LPS-198228 Create test for DDMDataProviderInstanceTableReferenceDefinition

### PR DESCRIPTION
**Pull request by gislayne-vitorino**
This is an update for: [LPS-198228](https://liferay.atlassian.net/browse/LPS-198228)

### CODE REVIEW

**Comment by gislayne-vitorino** [comment]
ci:test:publications

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/change/tracking/test/DDMDataProviderInstanceTableReferenceDefinitionTest.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+		DDMForm ddmForm = DDMFormTestUtil.createDDMForm("dataProviderName");
+
+		_ddmFormValues = DDMFormValuesTestUtil.createDDMFormValues(ddmForm);
+	}
+
+	@Override
+	protected CTModel<?> addCTModel() throws Exception {
+		return _ddmDataProviderInstanceLocalService.addDataProviderInstance(
+			TestPropsValues.getUserId(), group.getGroupId(),
```
Comment (david-truong) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/change/tracking/test/DDMDataProviderInstanceTableReferenceDefinitionTest.java:57]: can you make the same change I made here: https://github.com/liferay-publications/liferay-portal/pull/789

**Review by david-truong — CHANGES_REQUESTED** [review]

**Comment by gislayne-vitorino** [comment]
ci:test:sf

**Comment by gislayne-vitorino** [comment]
ci:test:publications

**Review by pei-jung — APPROVED** [review]

**Comment by pei-jung** [comment]
ci:forward

**Comment by david-truong** [comment]
ci:forward:force

**Review by david-truong — APPROVED** [review]

**Comment by david-truong** [comment]
ci:forward:force

**Pull request closed without merging**